apps/api/src/main.ts

```diff
@@ -6,12 +6,32 @@
 import { Logger } from '@nestjs/common'
 import { NestFactory } from '@nestjs/core'
 
+import { utilities as nestWinstonModuleUtilities, WinstonModule } from 'nest-winston'
+import * as winston from 'winston'
 import { AppModule } from './app/app.module'
 
 async function bootstrap () {
-  const app = await NestFactory.create(AppModule)
+  const app = await NestFactory.create(AppModule, {
+    logger: WinstonModule.createLogger({
+      transports: [
+        new winston.transports.Console({
+          format: winston.format.combine(
+            winston.format.timestamp(),
+            winston.format.ms(),
+            nestWinstonModuleUtilities.format.nestLike('Streaming Subs', {
+              colors: true,
+              prettyPrint: true
+            })
+          )
+        })
+        // other transports...
```

Comment:
Cool stuff! So if we want to forward logs to a s3 bucket or similar things to later index the logs this would be where we would add it?